### DISCUSSION TITLE
Categorizing Blog Posts

### DISCUSSION THREAD

**Discussion by blaccsmith** [blaccsmith/website#22] [Ideas]
We're adding blog posts to the site and are considering adding tags to help you better find relevant articles.

Currently, we've thought of `interviewing` `self care` and `productivity`.

🔍 What are some tags you would like to see on a blog?

**Comment by Genyus** [comment]
A bit late to this, but I my thoughts are your content should define your taxonomy and not the other way around.

What is the overall content strategy for your blog? Is it to be informational, or observational? Are you planning a broad or narrow range of topics? Frequent, bite-size articles, or in-depth essays?

Once you’ve decided on the type of content you want to create for your community, your tags should simply reflect and identify that.

**Reply by gt-codes** [reply]
We want blogs to be a space for people in the community to share their unique insights and experiences. Those can range from quick tips to in-depth articles! We don't want to limit the content based on tags, we're looking for suggestions to give us an idea on what type of content we could add.

**Reply by Genyus** [reply]
Ah, with you now. For me, I think a mix of technical how-tos/explainers, career advice (soft skills, interview techniques, etc) and maybe spotlights/interviews of community members would be cool

**Reply by gt-codes** [reply]
Those are great suggestions, I like it. Let me know if you're interested in any of these or helping us out with this!

**Reply by Genyus** [reply]
@Garrett1Tolbert Unfortunately, I'm already over-stretched so I can't commit to any additional active development, but would love to chat about the direction you're taking BLACC in and the potential for future collaboration with one of the projects I'm leading. Feel free to ping me in Discord if you'd like to discuss further.
